torch/_subclasses/fake_tensor.py

```diff
@@ -2051,6 +2081,65 @@ def go(t: object, real_t: Tensor) -> None:
                 else:
                     tree_map_(go, fake_out, real_out)
 
+                # check fake/real alias info
+                try:
+                    _check_alias_info(
+                        "Real tensor propagation found",
+                        real_out,
+                        (real_args, real_kwargs),
+                        fake_out,
+                        (args, kwargs),
+                    )
+                except AssertionError as exc:
+                    raise FakeRealKernelMismatchException(
```

Comment:
Why did we need a new exception type here?

Reply:
I guess we don't really.. I think I just misunderstood your comment above (on aliasing)